app/controllers/api/v1/work_experience_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::WorkExperienceController < ApplicationController
+    def create
+        @WorkExp = current_user.work_experiences.new(work_params)
+
+        if @WorkExp.save
+            render json: {status: 'Successful', message: 'Work Experience Added'}, status: 201
+        else
+            render json: {status: 'Failure', message: 'Something went wrong'}, status: 422
+        end
+    end
+
+    def index
+        @user_work = current_user.work_experiences
```

Comment:
- I think work experiences should be rendered in ascending or descending start_date order. I think handling that here will be more efficient than handling in the frontend.